### DISCUSSION TITLE
Cannot seem to download latest version?

### DISCUSSION THREAD

**Discussion by simondsmason** [hass-agent/HASS.Agent#212] [Q&A] [answered]
I know I must be doing something completely wrong but when I click on the HACs link from the documentation I get to the HASS Agent download and see this message:

 ⚠️❗**IMPORTANT ** ❗⚠️

You need at least HASS.Agent 2022.13.0 for this integration to work!

And when I download I only see this:

![image](https://github.com/user-attachments/assets/a74c6f74-2f67-4789-a58d-81bf659f0b92)

No option to download 2022.13.0 or later.  What am I missing?  Thanks.

**Comment by MolloPaul** [comment, marked as answer]
The HACs link from the documentation is incorrect and points to the old LAB02-Research Version.

The latest version needs to be installed as a custom repository see [https://github.com/hass-agent/HASS.Agent-Integration](url)

**Reply by simondsmason** [reply]
P.S.  The link provided gives a 404 - should be:

https://github.com/hass-agent/HASS.Agent-Integration
